src/app/pages/quickMaps/pages/baselineDetails/monthlyFood/monthlyFood.component.ts

```diff
@@ -53,24 +57,36 @@ export class MonthlyCardComponent implements OnInit, OnDestroy {
     // 'supplyUnit',
   ];
 
+  private loadingSrc = new BehaviorSubject<boolean>(false);
+  private errorSrc = new BehaviorSubject<boolean>(false);
+
+  private subscriptions = new Array<Subscription>();
+
   constructor(
     private currentDataService: CurrentDataService,
     private quickMapsService: QuickMapsService,
     private dialogService: DialogService,
     private cdr: ChangeDetectorRef,
-  ) {}
+    @Optional() @Inject(MAT_DIALOG_DATA) public data?: DialogData,
+  ) { }
 
   ngOnInit(): void {
-    this.resizeSub = this.resizeEvent.subscribe((widget) => {
```

Comment:
I might have missed this, but how are resize events managed for individual components now?

Reply:
You subscribe to a observable on the card e.g.:
https://github.com/micronutrientsupport/micronutrient-support-tool/blob/js_card_dialog_updates/src/app/pages/quickMaps/pages/baselineDetails/map-view/map-view.component.ts#L75